src/crawlee/_log_config.py

```diff
@@ -31,6 +35,44 @@
 _LOG_MESSAGE_INDENT = ' ' * 6
 
 
+def get_configured_log_level(configuration: Configuration) -> int:
+    verbose_logging_requested = 'verbose_log' in configuration.model_fields_set and configuration.verbose_log
+
+    if 'log_level' in configuration.model_fields_set:
+        if configuration.log_level == 'DEBUG':
+            return logging.DEBUG
+        if configuration.log_level == 'INFO':
+            return logging.INFO
+        if configuration.log_level == 'WARNING':
+            return logging.WARNING
+        if configuration.log_level == 'ERROR':
+            return logging.ERROR
```

Comment:
why not support all possible log levels?

Reply:
Less is more :shrug: Same as the comment higher up, I'd consider adding `CRITICAL`. Am I missing something?

Reply:
ok, adding just `CRITICAL` si good, thanks